apps/system/test/unit/notifications_test.js

```diff
@@ -144,6 +145,12 @@ suite('system/NotificationScreen >', function() {
     };
     navigator.mozL10n = MockL10n;
 
+    isDocumentHidden = false;
+    Object.defineProperty(document, 'hidden', {
+      configurable: true,
+      get: () => isDocumentHidden
```

Comment:
> Hmmm, then why even have a isDocumentHidden variable?

You can change the variable from a test (so that you can test other branches in your app code), but not from the app code.

If we have a setter, the app code could change it.

That's 2 very different ways to change the value.